cairo/tests/ethereum/cancun/test_transactions.py

```diff
@@ -58,3 +62,14 @@ def test_signing_hash_1559(self, cairo_run, tx: FeeMarketTransaction):
     def test_signing_hash_4844(self, cairo_run, tx: BlobTransaction):
         cairo_result = cairo_run("signing_hash_4844", tx)
         assert signing_hash_4844(tx) == cairo_result
+
+    @given(chain_id=..., tx=...)
+    def test_recover_sender(self, cairo_run, chain_id: U64, tx: Transaction):
+        try:
+            cairo_result = cairo_run("recover_sender", chain_id, tx)
```

Comment:
```suggestion
    def test_recover_sender(self, cairo_run_py, chain_id: U64, tx: Transaction):
        try:
            cairo_result = cairo_run_py("recover_sender", chain_id, tx)
```

Reply:
only if absolutely required - in that case, explicitly use `cairo_run_py`

Reply:
and create issue to migrate hints

Reply:
ah ok!